dss-api.yml

```diff
@@ -1496,7 +1496,7 @@ paths:
           required: false
           type: integer
           format: int32
-          minimum: 10
+          minimum: 2
```

Comment:
I think the minimum should stay 10 (or higher) to prevent our service from being overwhelmed by many tiny requests.

These values should not be changed to accommodate our testing strategies.

Reply:
done